src/management-system-v2/app/(dashboard)/[environmentId]/layout.tsx

```diff
@@ -146,6 +149,7 @@ const DashboardLayout: FC<PropsWithChildren<{ params: { environmentId: string }
         loggedIn={!!userId}
         userEnvironments={userEnvironments}
         layoutMenuItems={layoutMenuItems}
+        activeSpace={params.environmentId}
```

Comment:
You are right, the space id wasn't decoded, I am using `activeEnvironment` now for organisational spaces. For personal spaces we want to keep `"my"` though, because activeEnvironment would give us the id but this allows us to have one utility to get the space URL without needing to access the session. A future improvement could be to normalize all space handling to deal with objects `{ spaceId, isOrganization }` instead.